setup_environment.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+###################################################################
+# The purpose of this script is to download, install
+# and configure all files and programs required to
+# take part in developing ActivitySync (Android only).
+###################################################################
+
+CURRENT_USER=$USER
```

Comment:
Really ? I might be wrong, but in my case it works a bit differently. Try yourself.

```
# grzanka at gajewski-dell in ~ [12:56:04]
→ echo $USER     
grzanka

# grzanka at gajewski-dell in ~ [12:56:06]
→ sudo echo $USER
grzanka

```